circle.yml

```diff
@@ -4,11 +4,30 @@ jobs:
     working_directory: ~/tensorflow-serving-client
     docker:
       - image: circleci/python:3.6.1
-      - image: snormore/tensorflow-serving-http
+      - image: snormore/tensorflow-serving-http:1
         environment:
           MODEL_URL: https://storage.googleapis.com/tf-serving-docker-http-eae9e0c7-661d-4cca-836a-0433a8da44ba/imagenet/mobilenet_v1.tar.gz
-          MODEL_NAME: mobilenet_v1
-          MODEL_VERSION: 1
+          PORT: 9001
```

Comment:
> Pretty sure CI's not going to like this...

Circle only has a problem with connecting to `remote-docker-engine`, it's 👌  with doing it on images defined in here.